bitnami/external-dns/templates/deployment.yaml

```diff
@@ -220,14 +220,19 @@ spec:
             - --rfc2136-port={{ .Values.rfc2136.port }}
             - --rfc2136-zone={{ required "rfc2136.zone must be supplied for provider 'rfc2136'" .Values.rfc2136.zone }}
             - --rfc2136-min-ttl={{ .Values.rfc2136.minTTL }}
-              {{- if .Values.rfc2136.tsigKeyname }}
-            - --rfc2136-tsig-secret-alg={{ .Values.rfc2136.tsigSecretAlg }}
-            - --rfc2136-tsig-keyname={{ .Values.rfc2136.tsigKeyname }}
-                {{- if .Values.rfc2136.tsigAxfr }}
+              {{- if .Values.rfc2136.tsigAxfr }}
             - --rfc2136-tsig-axfr
-                {{- end }}
+              {{- end }}
+              {{- if .Values.rfc2136.rfc3645Enabled }}
+            - --rfc2136-gss-tsig
+            - --rfc2136-kerberos-realm={{ required "rfc2136.kerberosRealm must be supplied for provider `rfc2136` if `rfc2136.rfc3645Enabled` is true." .Values.rfc2136.kerberosRealm }}
```

Comment:
The `values.yaml` file says the realm is optional, but it shows as required here. Which one is correct?

If it's required, this validation fits the `_helpers.tpl` file better.

Reply:
It is required if rfc3645Enabled is true -- is there a better way to express that?

Reply:
Done in latest commits